### DISCUSSION TITLE
Can I use VS Code simultaneously from two/three different locations?

### DISCUSSION THREAD

**Discussion by progettoautomazione** [coder/code-server#4741] [Q&A] [answered]
Hi everyone, I've installed VS Code 4 on my Ubuntu server, behind reverse proxy. I'm wondering if I'll have troubles providing the access to this VS Code instance to my colleagues, so that we can simultaneously code on the same VS Code workspace. Do you see some problems in doing this? Of course all files will be tracked as modified/created by the same user, that's ok. I'm more thinking/asking about technical issues that might case problems in using the tool.
Thanks

**Comment by jsjoeio** [comment, marked as answer]
Hmm.. you wouldn't be able to simultaneously code on the same VS Code workspace. You'd have to use some type of collaboration tool in order to do that. See [the docs](https://coder.com/docs/code-server/latest/collaboration) for more information.

**Comment by progettoautomazione** [comment]
we wouldn't be able to work on the same workspace even if we work on
different files? Thanks for the link, I'll check those tools

**Reply by jsjoeio** [reply]
TBH, I have not tried it...so you could and report back?
